generator/src/generator/namespace/mod.rs

```diff
@@ -76,17 +76,22 @@ impl<'ns, 'c> NamespaceGenerator<'ns, 'c> {
         }
     }
 
+    fn is_unix_only(&self) -> bool {
+        self.ns.contains_fds()
```

Comment:
Just because an extension contains some request with FDs, that does not make the whole thing unix-only. Especially RandR does not seem so unix-only (I had to look up the one use of FDs in RandR).

Is this really necessary? Doesn't the existing `FdContainer` code for non-unix also work for non-std?

Reply:
My bad, that’s an artifact of when I made FDs Unix-only. It should be named "is_std_only"

Reply:
Still, RandR is not really std-only. Only a single request in there uses FDs and that could be worked around somehow else, I think.

Reply:
Fair enough, I guess I could be more granular and only gate the items that use FDs, not the module.

Reply:
Is that even needed? Why isn't the existing gating in `x11rb-protocol/src/utils.rs` enough? It was so far used to solve the same problem for `cfg(unix)` instead of `cfg(std)`.